src/community/resources/index.md.njk

```diff
@@ -0,0 +1,103 @@
+
+A curated list of community developed resources that can help you use the GOV.UK Design System in your chosen design tool or development environment. 
+
+The Design System team is not responsible for these resources and is unable to provide support for them. If you have support or feature requests please contact the owner.
+
+
+## Create flow diagrams
+
+[GOV.UK Design System Flow Diagrams](https://github.com/dashouse/govuk-design-system-flow-diagrams)⁠ -  
+A Sketch file for creating flow diagrams of GOV.UK services
+
+[GOV Flow](https://github.com/charlesrt/gov-flow)⁠ -  
+A Sketch file for creating flow diagrams of GOV.UK services
+
+[GOV Flow for Google Drawings](https://github.com/charlesrt/gov-flow)⁠ -  
+Flow diagram templates for Google drawings, based on GOV Flow
+
+https://www.beatnic.co.uk/2019/10/04/google-drawing-template-gov-flow-kit/
+
+[GOV.UK Userflow Icons](https://github.com/alirawashdeh/govuk-userflow-icons) -  
+SVG images for creating flow diagrams of GOV.UK services
+
+## Create prototypes and wireframes
+
+[govuk-design-system-balsamiq](https://github.com/enoranidi/govuk-design-system-balsamiq)⁠ -  
+Balsamiq wireframes based on the GOV.UK Design System
+
+[Sketch Wireframing Kit](https://github.com/abbott567/sketch_wireframing_kit)⁠ -  
+Sketch wireframes based on the GOV.UK Design System
+
+## Build front ends
+
+### C#
+
+[GOV.UK Design System in C#/ASP.Net Core](https://github.com/cabinetoffice/govuk-design-system-dotnet)⁠ -  
+A collection of GOV.UK Frontend components built in C# / ASP.Net Core
+
+
+### Node.js
+
+[Hapi GOV.UK examples](https://github.com/DEFRA/hapi-govuk-examples)⁠ -  
+An introduction to building a hapi service using GOV.UK Frontend
+
+
+### PHP
+
+[Drupal theme](https://www.drupal.org/project/govuk_design_system)⁠ -  
+A Drupal theme that uses GOV.UK Frontend
+
+
+### Python
+
+[Python Microservice](https://github.com/willp-bl/python-microservice)⁠ -  
+A Flask/Python implementation of GOV.UK Frontend
+
+[GOV.UK Frontend Jinja ](https://github.com/alphagov/govuk-frontend-jinja)⁠ -  
+Classes and modules for using GOV.UK Frontend in your Jinja2-powered Python app
+
+[gov.js](https://gist.github.com/andymantell/8b1bb037341f38990a4d019894abd83e)⁠ -  
+A Gulp task to convert GOV.UK Frontend Nunjucks macros to Jinja2
+
+### R
+
+[Govdown](https://github.com/ukgovdatascience/govdown)⁠ -  
+A GOV.UK Design System theme for R Markdown
+
+### Ruby on Rails
+
+[GOV.UK Frontend Rails](https://github.com/dxw/dxw_govuk_frontend_rails)⁠ -  
+A Ruby Gem that adds the GOV.UK Frontend to Rails apps
+
+[GOV.UK Rails Boilerplate](https://github.com/DFE-Digital/govuk-rails-boilerplate)⁠ -  
+A complete Rails starter app using GOV.UK Frontend
+
+[GOV.UK Design System Formbuilder](https://github.com/DFE-Digital/govuk_design_system_formbuilder) -  
+A Rails form builder using GOV.UK Frontend
+
+## Write code
+
+[GOV.UK Design System snippets Atom package](https://atom.io/packages/govuk-design-system-snippets)⁠ -  
```

Comment:
There's duplication in each description here, which is a warning sign this could be made simpler. I think it'd be better to move away from the list format here, for example:

You can download GOV.UK Frontend Nunjucks macro snippets for:

- [Atom]
- [Visual Studio Code]
- [other code editors]

I wonder whether this format could be applied to previous sections too - which would help avoid needing to use the title and the description in each case. But that's probably for another day.

Reply:
This one is tricky.

The first two can be installed directly from the Atom or VS Code code editors. The 3rd one is ALSO for Atom and VS Code, but you need to download the repo first and then install them manually.

Reply:
For now I'm removing the 3rd resource, as it seems to duplicate the other 2